docs/products/m3db/howto/write-python.rst

```diff
@@ -0,0 +1,48 @@
+Write Data to M3DB with Python
+------------------------------
+
+This example writes some data to an M3DB service from Python, making use of the InfluxDB library.
+
+Variables
+'''''''''
+
+These are the placeholders you will need to replace in the code sample:
+
+=================       =======================================================
+Variable                Description
+=================       =======================================================
+``SERVICE_HOST``        Service hostname, found on the service overview page
+``SERVICE_PORT``        Service port number, found on the service overview page
+``AVNADMIN_PASS``       Password for the ``avnadmin`` user
```

Comment:
Should we say that is the default user? "Password for the **default** ``avnadmin`` user"

Reply:
Fixed